wagtail/snippets/tests/test_viewset.py

```diff
@@ -769,6 +770,47 @@ def test_boolean_column(self):
         )
 
 
+class TestRelatedFieldColumns(BaseSnippetViewSetTests):
+    model = SnippetChooserModel
+
+    def setUp(self):
+        super().setUp()
+        url = "https://example.com/free_examples"
+        self.advert = Advert.objects.create(url=url, text="Free Examples")
+        self.ffs = FullFeaturedSnippet.objects.create(
+            text="test snippet with custom icon"
+        )
+        self.scm = self.model.objects.create(advert=self.advert, full_featured=self.ffs)
+        self.no_ffs_chooser = self.model.objects.create(advert=self.advert)
+
+    def test_snippetchoosermodel_related_fields(self):
```

Comment:
We're not testing the `SnippetChooserModel`, so it's better to use a more descriptive name e.g.

```suggestion
    def test_single_level_relation(self):
```